pkg/util/binding.go

```diff
@@ -87,6 +87,12 @@ func DivideReplicasByTargetCluster(clusters []workv1alpha2.TargetCluster, sum in
 
 // MergeTargetClusters will merge the replicas in two TargetCluster
 func MergeTargetClusters(old, new []workv1alpha2.TargetCluster) []workv1alpha2.TargetCluster {
+	switch {
```

Comment:
IMOP, this logic may seem a little redundant, after all, the following logic already overrides this judgment.

Reply:
This is the short path.
If we need a short patch depends on :
1. How often does the short patch hit? 
2. The following logic is complicated and most cases don't need to be involved.

cc @Garrybest for comment

Reply:
When first scheduled, the `old` cluster must be empty. So I think this patch is useful.

Reply:
ok. here we go.